packages/tools/src/stateManagement/segmentation/polySeg/Surface/convertLabelmapToSurface.ts

```diff
@@ -0,0 +1,60 @@
+import { Types, cache, eventTarget, triggerEvent } from '@cornerstonejs/core';
+import { getWebWorkerManager } from '@cornerstonejs/core';
+import {
+  LabelmapSegmentationData,
+  LabelmapSegmentationDataStack,
+  LabelmapSegmentationDataVolume,
+} from '../../../../types/LabelmapTypes';
+import { computeVolumeSegmentationFromStack } from '../../convertStackToVolumeSegmentation';
+import { Events } from '../../../../enums';
+
+const workerManager = getWebWorkerManager();
+
+export async function convertLabelmapToSurface(
```

Comment:
At least a short docs here please

Reply:
Need to know that this will fail if the volume segmentation from stack fails - least that is what it appear to happen to me.  Certainly that is an important bit of information to know when calling this against stack data.